seqr/views/utils/file_utils.py

```diff
@@ -54,7 +65,7 @@ def parse_file(filename, stream):
 
         return rows
 
-    elif filename.endswith('.json'):
+    elif filename.endswith('.json') and not iter_file:
         return json.loads(stream.read())
 
     raise ValueError("Unexpected file type: {}".format(filename))
```

Comment:
is it worth worth adding the value of `iter_file` in this message? there's a possible case now where the file is a valid format but `iter_file` is True and this error is raised, in this case the error message would be incorrect

Reply:
agreed